src/components/companies/companies.js

```diff
@@ -1,28 +1,29 @@
 import React, {Component} from 'react';
 
 import {CompaniesList} from '../companies-list/companies-list.js';
-import CompaniesAddForm from '../companies-add-form/companies-add-form.js';
-import SearchPanel from 'components/search-panel/search-panel.js';
+import {SearchPanel} from 'components/search-panel/search-panel.js';
+import { CompanyForm } from 'components/company-form/company-form.js';
```

Comment:
{A} or { A } - pick one style and use it everywhere

Reply:
done